.code-samples.meilisearch.yaml

```diff
@@ -477,3 +477,9 @@ faceted_search_walkthrough_facets_distribution_1: |-
   $ curl --get 'http://localhost:7700/indexes/movies/search' \
       --data-urlencode 'q=Batman' \
       --data-urlencode 'facetsDistribution=["genres"]'
+post_backup_1: |- 
+  $ curl -X POST 'http://localhost:7700/backup',
+get_backup_status_1: |- 
+  $ curl -X GET 'http://localhost:7700/backup/:backup_uid/status',
+get_backup_1: |- 
```

Comment:
It seems like that for information you should use `/:backup_ui/status` and not just `/:backup`.
Nevertheless since these code samples id's are used to help developers understand what is expected from this examples, either `get_backup` or `download_backup` is as clear for me with a small preference for `download_backup` or `fetch_backup` since [fetch](https://www.computerhope.com/jargon/f/fetch.htm#:~:text=Fetch%20describes%20the%20process%20of,alternate%20location%20or%20displaying%20it.) seems to be a good fit for the use case.